app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -81,5 +81,9 @@ def set_workflow_flags
     def auth
       ProviderAuthorisation.new(actor: current_provider_user)
     end
+
+    def state_store_key
+      "#{ProviderApplicationsFilter::STATE_STORE_KEY}_#{current_provider_user.id}"
```

Comment:
should we pass this into our generic cache key generator so it adds the SHA and feature flag updates incase we change the filters?